env/WCOSS2.env

```diff
@@ -0,0 +1,232 @@
+#!/bin/ksh -x
+
+if [ $# -ne 1 ]; then
+
+    echo "Must specify an input argument to set runtime environment variables!"
+    echo "argument can be any one of the following:"
+    echo "anal fcst post vrfy metp"
+    echo "eobs eupd ecen esfc efcs epos"
+    echo "postsnd awips gempak"
+    exit 1
+
+fi
+
+step=$1
+
+# WCOSS2 information
+export launcher="mpiexec -l"
+export mpmd="--cpu-bind verbose,core cfp"
+
+export npe_node_max=128
+
+# Configure MPI environment
+export MPI_LABELIO=YES
+export MP_STDOUTMODE="ORDERED"
+export KMP_STACKSIZE=2048M
+export KMP_AFFINITY=scatter
```

Comment:
This is fine for now, but we should ask the system owner if these are advantageous or not needed on this machine.

Reply:
Definitely...these are what is used on the Dells right now so I need inputs from others on what (if any) to set here.